sys/shell/shell.c

```diff
@@ -256,6 +256,9 @@ static int readline(char *buf, size_t size)
         else {
             *line_buf_ptr++ = c;
             _putchar(c);
+#ifdef MODULE_NEWLIB
+            fflush(stdout);
+#endif
```

Comment:
about shell you are right, formally it is not related to this cpu, the important thing is that if it is a bug then it has to be pulled into.

For doxygen and Makefile they also are not part of this PR, but this PR is just a workaround ... the goal is to have all the files into the final merge and I've created the PR in the most easy way for me.

I hope this doesnt hurt too much ...